pkg/service/rulesvc/get.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/common-fate/common-fate/pkg/storage"
 )
 
-func (s *Service) GetRule(ctx context.Context, ID string, user *identity.User, isAdmin bool) (*rule.AccessRule, error) {
+func (s *Service) GetRule(ctx context.Context, ID string, user *identity.User, isAdmin bool) (*rule.AccessRuleWithCanRequest, error) {
```

Comment:
Functionality looks good, can you please rename this struct so it's the same as some of our other services for consistency
```suggestion
func (s *Service) GetRule(ctx context.Context, ID string, user *identity.User, isAdmin bool) (*rule.GetAccessRuleResponse, error) {
```